CONTRIBUTING.md

```diff
@@ -28,7 +28,10 @@ point to the `rules_jvm` folder.
 
 ## Releasing
 
-1. Determine the next release version, following semver if possible/
-2. Tag the repo and push it (or create a tag in GH UI).
-3. Watch the automation run on GitHub actions.
+1. Determine the next release version, following semver if possible.
+2. Tag the repo and push it (e.g. `git tag v1.2.3`), or create a tag in GitHub UI.
+3. Watch the automation run via the [GitHub action](.github/workflows/release.yml).
+4. Once the GitHub release has been created, you can manually edit it.
+5. Update the Bazel Central Registry's contrib_rules_jvm module, as in [this PR][]
```

Comment:
@shs96c do you know how you configured [publish-to-bcr-bot](https://github.com/apps/publish-to-bcr-bot) for https://github.com/bazelbuild/bazel-central-registry/pull/3770 ?

When i pushed https://github.com/bazel-contrib/rules_jvm/releases/tag/v0.29.0 nothing got triggered. I'm guessing maybe it's because you had already configured the [Legacy GitHub App](https://github.com/bazel-contrib/publish-to-bcr#legacy-github-app) method for the bot, and we now need to follow the new [Setup steps](https://github.com/bazel-contrib/publish-to-bcr#setup). If so then I'll leave these docs as-is, and we can look at said automation in a follow-up PR.